tests/test-sources/plugins/none-ls.nix

```diff
@@ -21,34 +21,45 @@
       lsp-format.enable = true;
       none-ls = {
         enable = true;
-        enableLspFormat = true;
+        # This is implied:
+        # enableLspFormat = true;
```

Comment:
Just remove it entirely then, no ?

Reply:
Could do, but since this is kind of a weird option (`default = plugins.lsp-format.enable`) and also an option we might remove/rework in the future, I thought it was useful to note the implication.

Up to you.

Reply:
No it's fine like this.